integration/api/backend_jail_test.go

```diff
@@ -486,88 +486,89 @@ func (s *BackendTestSuite) TestJailWhisper() {
 			`,
 			true,
 		},
-		{
-			"test 4: encrypted anonymous message (From == nil && To != nil)",
-			`
-				var identity = '` + accountKey2Hex + `';
-				if (!shh.hasKeyPair(identity)) {
-					throw 'idenitity "` + accountKey2Hex + `" not found in whisper';
-				}
-
-				var topic = makeTopic();
-				var payload = '` + whisperMessage4 + `';
-
-				// start watching for messages
-				var filter = shh.newMessageFilter({
-					privateKeyID: identity,
-					topics: [topic],
-				});
-
-				// post message
-				var message = {
-					ttl: 20,
-					powTarget: 0.01,
-					powTime: 20,
-					topic: topic,
-					pubKey: identity,
-			  		payload: web3.toHex(payload),
-				};
-
-				var sent = shh.post(message)
-				if (!sent) {
-					throw 'message not sent: ' + JSON.stringify(message);
-				}
-
-				var filterName = '` + whisperMessage4 + `';
-				var filterId = filter.filterId;
-				if (!filterId) {
-					throw 'filter not installed properly';
-				}
-			`,
-			true,
-		},
-		{
-			"test 5: encrypted signed response to us (From != nil && To != nil)",
-			`
-				var identity1 = '` + accountKey1Hex + `';
-				if (!shh.hasKeyPair(identity1)) {
-					throw 'idenitity "` + accountKey1Hex + `" not found in whisper';
-				}
-				var identity2 = '` + accountKey2Hex + `';
-				if (!shh.hasKeyPair(identity2)) {
-					throw 'idenitity "` + accountKey2Hex + `" not found in whisper';
-				}
-				var topic = makeTopic();
-				var payload = '` + whisperMessage5 + `';
-				// start watching for messages
-				var filter = shh.newMessageFilter({
-					privateKeyID: identity1,
-					sig: identity2,
-					topics: [topic],
-				});
-
-				// post message
-				var message = {
-				  	sig: identity2,
-				  	pubKey: identity1,
-				  	topic: topic,
-				  	payload: web3.toHex(payload),
-					ttl: 20,
-					powTime: 20,
-					powTarget: 0.01,
-				};
-				var sent = shh.post(message)
-				if (!sent) {
-					throw 'message not sent: ' + message;
-				}
-				var filterName = '` + whisperMessage5 + `';
-				var filterId = filter.filterId;
-				if (!filterId) {
-					throw 'filter not installed properly';
-				}
-			`,
-			true,
-		},
+		// @TODO(adam): fix in #336
```

Comment:
Do we not require this tests anymore?

Reply:
We need it but it's broken :( Take a look at #336.

Reply:
Oh shame. Alright, thanks.